code/controllers/subsystem/vote.dm

```diff
@@ -180,7 +180,7 @@ SUBSYSTEM_DEF(vote)
 			if("map")
 				// Find target map.
 				var/datum/map/top_voted_map
-				for(var/x in subtypesof(/datum/map))
+				for(var/x in subtypesof(/datum/map) - /datum/map/metastation) //remove this Meta removal once we find a fix to Meta/another Map. - MattTheFicus
```

Comment:
This is incredibly snowflake and should be done on the map datum itself with a voteable var.

Reply:
Desnowflake'd via datum variable.